src/main/java/edu/kis/powp/jobs2d/command/gui/ComplexCommandEditorWindow.java

```diff
@@ -0,0 +1,260 @@
+package edu.kis.powp.jobs2d.command.gui;
+
+import edu.kis.legacy.drawer.panel.DrawPanelController;
+import edu.kis.legacy.drawer.shape.LineFactory;
+import edu.kis.powp.appbase.gui.WindowComponent;
+import edu.kis.powp.jobs2d.command.CompoundCommand;
+import edu.kis.powp.jobs2d.command.DriverCommand;
+import edu.kis.powp.jobs2d.command.OperateToCommand;
+import edu.kis.powp.jobs2d.command.SetPositionCommand;
+import edu.kis.powp.jobs2d.command.modifier.model.Point;
+import edu.kis.powp.jobs2d.drivers.adapter.LineDriverAdapter;
+import edu.kis.powp.jobs2d.features.CommandsFeature;
```

Comment:
Lets try reducing some dependencies. Start from CommandFeature.

Reply:
Imports shouldn't be replaced using wildcard * because of rules of clean code and performance

Reply:
Using a wildcard * does not remove the dependency. Please try rewriting the code to avoid dependencies to CommandFeature and possibly DriverCommandManager. Notice, that now we have CommandManager interfaces introduced (recent merged PR-s)